### ISSUE TITLE
1.1.2 rules cause an infinite loop Layout/ArgumentAlignment -> Layout/HashAlignment

### ISSUE THREAD

**Issue by jarkko** [testdouble/standard#306]
The latest dependabot update to standardrb 1.1.2 / rubocop 1.17.0 caused this `InfiniteCorrectionLoop` exception when using the `--fix` option:

```
Parallel::UndumpableException: RuboCop::Runner::InfiniteCorrectionLoop: Infinite loop detected in year_end_tax_data_importer_spec.rb and caused by Layout/ArgumentAlignment -> Layout/HashAlignment
```

The code in question looks like this:

```ruby
  describe "import_for_year!" do
    before do
      stub_request(:get, "https://www.ictax.admin.ch/extern/api/download/1932147/kursliste_2020.zip")
        .to_return(status: 200,
                   body: File.read(Rails.root.join("spec", "fixtures", "kursliste_2019.xml.zip")), headers: {})
    end
…
```

So the issue is the alignment of the `body` key. However, this is what the previous version auto-corrected the code to (and there are loads of these in the codebase).

Without the `--fix` arg, the output looks like this:

```
spec/lib/year_end_tax_data_importer_spec.rb:27:20: C: [Correctable] Layout/ArgumentAlignment: Use one level of indentation for arguments following the first line of a multi-line method call.
                   body: File.read(Rails.root.join("spec", "fixtures", "kursliste_2019.xml.zip")), headers: {})
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

1 file inspected, 1 offense detected, 1 offense auto-correctable
```

The only way I can fix this is by dropping the whole hash args on a separate line from the starting parenthesis. However, this sounds like something that is actually auto-correctable (as the output implies), and anyway guess it shouldn't cause an infinite correction loop.

**Comment by searls** [comment]
Whenever I see an `InfiniteCorrectionLoop`, my first reaction to verify it isn't a bug in Rubocop itself. @koic, what do you think?

**Comment by koic** [comment]
@searls Thank you for mentioning. This is a bug of RuboCop. I will proceed with the investigation to solve this issue.

**Comment by koic** [comment]
I opened https://github.com/rubocop/rubocop/pull/9897.

**Comment by koic** [comment]
And RuboCop 1.18.1 has been released with this fix.

**Comment by jmkoni** [comment]
@koic thank you!

**Comment by jmkoni** [comment]
Released with 1.1.3!
